weblate/trans/models/change.py

```diff
@@ -1038,6 +1112,14 @@ def show_unit_state(self):
             self.ACTION_SUGGESTION_CLEANUP,
         }
 
+    def get_uuid(self) -> uuid.UUID:
+        """Return uuid for this change."""
+        return uuid.uuid5(uuid.NAMESPACE_OID, f"{self.action}.{self.id}")
```

Comment:
This is really not a OID. I think it's better to define the own namespace than using a standard one in a non-standard way.